### ISSUE TITLE
Does it has support for JavaScript-powered sites?

### ISSUE THREAD

**Issue by joanlopez** [Amitbhave/flutter-simple-url-preview#1]
Hi there,

Precisely some weeks ago I was researching for a library intended to show a URL preview, but no one of the ones I tested did work.

So, I decided to develop my own URL preview component. However, I realised that, at the end, which was the problem was the URL I was using for tests. I was trying to preview a link to a tweet (Twitter).
The key here is that, when you gets the HTML content of a Twitter page, it returns a simple HTML telling you should enable JavaScript. So, without JavaScript, the factory can't find the Open Graph data.

So, is it supported?

Thanks,
Joan

**Comment by Amitbhave** [comment]
@joanlopez indeed this is not currently supported in the package.. Thanks for raising!

**Comment by joanlopez** [comment]
Nope, unfortunately I didn't find any way to do it. In fact, I'd say that I'd developed my own widget if I'd found it.

However, I neither explored it a lot. So, it might bé interesting to look for it. Indeed, I'd really like to contribute to this
